src/CSnakes.SourceGeneration/PythonStaticGenerator.cs

```diff
@@ -137,14 +135,16 @@ public void Dispose()
                     {{methods.Select(m => m.Syntax).Compile()}}
                 }
             }
-            public interface I{{pascalFileName}} : IDisposable
```

Comment:
IS the removal of `IDisposable` intentional?

Reply:
It's inherited indirectly via `IReloadableModuleImport`.